requirements.txt

```diff
@@ -9,13 +9,13 @@ channels_redis==2.4.0
 django-extra-fields==0.9
 pillow==6.2.0
 celery==4.2.1
-gunicorn==19.9.0
+gunicorn==20.0.4
 # flower==0.9.2
 flower==0.9.2
 # Tornado here is just to fix a flower problem..
 tornado>=4.2.0,<6.0.0
 urllib3<1.25,>=1.21.1
-uvicorn==0.8.4
+uvicorn==0.11.2
```

Comment:
in build, this complains about needing websocket==8

Reply:
bumped to `8.1` to match compute worker